config/config.go

```diff
@@ -153,12 +160,20 @@ func (c BlockManagerConfig) Validate() error {
 		return fmt.Errorf("batch_submit_max_time must be greater than max_idle_time")
 	}
 
+	if c.BatchSubmitMaxTime > MaxBatchSubmitMaxTime {
```

Comment:
should we also have a mininum for that?

Reply:
i think having a minimum value equivalent to max_idle_time is enough.